pages/partner-with-us.js

```diff
@@ -1,12 +1,84 @@
+import { BLOCKS } from '@contentful/rich-text-types';
 import React from 'react';
+import { createClient } from 'contentful';
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+
+import Card, {
+  CardAddress,
+  CardParagraph,
+  CardTitle,
+} from '../components/Card';
 
-import ElectronicsDropOffCard from '../components/ElectronicsDropOffCard';
 import PrimaryLayout from '../components/PrimaryLayout';
 
-export default function PartnerWithUs() {
+const options = {
+  renderNode: {
+    [BLOCKS.PARAGRAPH]: (_, children) => (
+      <CardParagraph>{children}</CardParagraph>
+    ),
+    [BLOCKS.UL_LIST]: (_, children) => (
+      <ul className="list-disc text-left pl-6">
+        {children.map((item) => (
+          <li key={item.key}>{item.props.children[0].props.children[0]}</li>
+        ))}
+      </ul>
+    ),
+  },
+};
+
+export async function getServerSideProps() {
+  const client = createClient({
+    environment: process.env.CONTENTFUL_ENVIRONMENT,
+    space: process.env.CONTENTFUL_SPACE_ID,
+    accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
+  });
+
+  const res = await client.getEntries({
+    content_type: 'pageWithPartnerCards',
+    'fields.slug': 'partnerwithuscards',
+  });
+
+  return {
+    props: {
+      page: res.items[0],
+    },
+  };
+}
+
+export default function PartnerWithUs({
+  page: {
+    fields: { partnerCards },
+  },
+}) {
   return (
     <PrimaryLayout>
-      <ElectronicsDropOffCard />
+      {partnerCards.map((partnerCard) => {
+        const {
+          fields: {
+            image: {
+              fields: {
+                file: { url },
+              },
+            },
+            description: rawRichTextField,
+            email: emailRichTextFeild,
+            title,
+          },
+          sys: { id },
+        } = partnerCard;
+
+        return (
+          <Card backgroundImageSource={`https:${url}`} key={id}>
+            <CardTitle>{title}</CardTitle>
+            {documentToReactComponents(rawRichTextField, options)}
+            <CardAddress label="Email us to learn more at:">
+              <a style={{ color: 'yellow' }}>
```

Comment:
Please do not use inline styles.

Reply:
Thanks for noticing this @marekrozmus. Here the inline style was from pre-existing code in the repo that was just moved over, but I guided @ozer619 and @Sahil-101 on removing it. 🙂